### PR TITLE
added Product Page

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
Product Page:
Screenshot (667)

### CODE REVIEW

File: Product-page/merch.css [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    flex-direction: column;
+    align-items: center;
+    font-family:Arial, Helvetica, sans-serif ;
+    background-image: linear-gradient(black, purple);
+}
+.header{
+    height: 7vh;
+    width: 80%;
+    display: flex;
+    background-color: #000000;
```
Comment (sharmayatendra) [inline Product-page/merch.css:27]: avoid using hard coded color value

File: Product-page/merch.css [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    color: white;
```
Comment (sharmayatendra) [inline Product-page/merch.css:28]: same here use variables for color

File: Product-page/merch.css [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    flex-direction: column;
+    gap: 1rem;
+    overflow-y: hidden;
+}
+.sidebar-head{
+    display: flex;
+    justify-content: space-between;
+    padding:1rem 1rem 0rem 1rem;
+}
+.sidebar-head span{
```
Comment (sharmayatendra) [inline Product-page/merch.css:111]: try to target class instead of targeting the element

File: Product-page/merch.css [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+    max-inline-size: 100%;
+    block-size: auto;
+    aspect-ratio: 4/3;
+    object-fit: contain/cover;
+    object-position: top center;
+    height: 100%;
+    width: 100%;
+    
+    object-fit: cover;
```
Comment (sharmayatendra) [inline Product-page/merch.css]: why used 2 times object-fit property?

File: Product-page/merch.html [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+                    <div class="sidebar-head-btn">
+                        ClearAll
+                    </div>
+                </div>
+                <div class="sidebar-divide"></div>
+                <div class="sidebar-items">
+                    <div class="sidebar-subhead">SORT</div>
+                    <div class="sidebar-content">
+                        <div class="sidebar-cont-item">
+                            <input type="radio" name="sort" id="">
```
Comment (sharmayatendra) [inline Product-page/merch.html:40]: why id=""?

File: index.html [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+                        <div class="buy-btn">
+                            <button>Buy Now</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="head-cards">
+                <div class="head-card-item active">
+                    <span>Dying light 2</span>
+                    <img src="./Assets/dying-light-2.jfif" alt="">
```
Comment (sharmayatendra) [inline index.html:48]: use proper alt attribute for images

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,374 @@
...
+    width: 100%;
+    height: 45rem;
+    border: solid;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background:linear-gradient(transperant, black, 30) url("./Assets/dying-light-2.jfif");
+    background-repeat: no-repeat;
+    background-size: cover;
+    filter: blur(); */
```
Comment (sharmayatendra) [inline styles.css:24]: if not needed remove commented code

**Review by sharmayatendra — APPROVED** [review]
can work upon suggested changes. good work.

**Review by md-amir-gauhar — APPROVED** [review]
Just follow the suggestions given above. Good work though

**Comment by HrishikeshMahalle** [comment]
> can work upon suggested changes. good work.

Thank you @sharmayatendra Will surely look into al the changes since some changes are in main file, i can't update them before merging,  so keeping all the points noted i'll  do those changes . Thank you for such a Awesome PR.

**Pull request merged**